libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/UIDevice+SFHardware.m

```diff
@@ -599,20 +599,20 @@ - (NSString *) macaddress
 }
 
 - (UIInterfaceOrientation)interfaceOrientation {
```

Comment:
it looks like the comment in the header for this method mentions falling back on the status bar, could you update that too?

Reply:
Yes, updated.